{{cookiecutter.project_name}}/.github/workflows/test.yaml

```diff
@@ -48,14 +63,28 @@ jobs:
       - name: Install uv
         uses: astral-sh/setup-uv@v5
         with:
+          python-version: ${{ matrix.env.python }}
           cache-dependency-glob: pyproject.toml
+      - name: create hatch environment
+        run: uvx hatch env create ${{ matrix.env.name }}
       - name: run tests using hatch
         env:
           MPLBACKEND: agg
           PLATFORM: ${{ matrix.os }}
           DISPLAY: :42
-        run: uvx hatch test --cover --python ${{ matrix.python }}
+        run: uvx hatch run ${{ matrix.env.name }}:run-cov
       - name: generate coverage report
-        run: uvx hatch run hatch-test.py${{ matrix.python }}:coverage xml
+        run: uvx hatch run ${{ matrix.env.name }}:coverage xml
       - name: Upload coverage
         uses: codecov/codecov-action@v4
+
+  check: # https://github.com/re-actors/alls-green#why
+    if: always()
+    needs:
+      - get-environments
+      - test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: re-actors/alls-green@release/v1
```

Comment:
I suggest to use a more descriptive name, currently it just shows up as "check" 

![image](https://github.com/user-attachments/assets/4a6a7fc1-ff33-4496-9839-bbb2649235bd)


```suggestion
      - uses: re-actors/alls-green@release/v1
        name: Tests pass in all hatch environments
```

Reply:
Hmm, one could also add other jobs to it, but I guess then people can also change the name